nft/nfts_body.yaml

```diff
@@ -99,3 +99,10 @@ refreshCache:
   schema:
     type: boolean
     default: false
+block:
+  name: block
+  description: String - The block number a query should be run at. Used to fetch historical / snapshot data
```

Comment:
the description only needs to be defined in one place! so we can define it here and read it into nfts.yaml or define it in nfts.yaml and remove the description from nfts_body.yaml

Moving forward I'm planning on consolidating the NFT API methods into one file (nfts.yaml) so we can choose either place for now.

Reply:
Ah I see, I'm going to move it to nfts.yml because it can be a different description per api, thanks!